client/webserver/locales/en-us.go

```diff
@@ -165,4 +165,6 @@ var EnUS = map[string]string{
 	"Restoration Seed":       "Restoration Seed",
 	"Restore from seed":      "Restore from seed",
 	"Import Account":         "Import Account",
+	"no_wallet":              "no wallet",
+	"create_a_x_wallet":      "Create a {{.Info.Name}} Wallet", // .Info.Name will be an asset symbol like DCR
```

Comment:
To document the reasoning for this, in portuguese we use that different from english, so we say "criar uma carteira decred" like it was: "create a wallet decred".  So doing ` <button data-action="create">[[[Create a]]] {{.Info.Name}} [[[Wallet]]]</button>` created a challenge.
My suggestion was simply to use go html template xpressions in the locale strings and let them find their way in to the localized_html files, which they do.  `<button data-action="create">Criar uma carteira {{.Info.Name}}</button>`

Reply:
Also we can now get rid of the `"Create a"` map entry, correct?